server/app/services/mongo_pubsub.rb

```diff
@@ -51,28 +62,54 @@ def send_message(data)
 
   attr_accessor :collection
 
+  finalizer :finalize
+
+  # @return [Array<Subscription>]
+  def subscriptions
+    @@subscriptions ||= []
```

Comment:
This will be shared if the class is inherited from. A safer way to implement might be:

```ruby
def self.subscriptions
  @subscriptions ||= []
end

def subscriptions
  self.class.subscriptions
end
```

Also as this is used in a multi-threaded setup, I believe this should be some kind of thread-safe primitive, as seen in [Celluloid docs](https://github.com/celluloid/celluloid/wiki/Exclusive#all-methods)

Reply:
Per design only the actor thread should be accessing the `subscriptions`, so it shouldn't need to be threadsafe.